features/features.go

```diff
@@ -670,4 +670,11 @@ var (
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						enhancementPR("https://github.com/openshift/enhancements/pull/1697").
 						mustRegister()
+	FeatureGateKMSv2 = newFeatureGate("KMSEncryptionProvider").
```

Comment:
Need a new line above this

Also, please be consistent about the naming
```suggestion
	
	FeatureGateKMSEncryptionProvider = newFeatureGate("KMSEncryptionProvider").
```

Reply:
thanks, fixed both (new line + naming).